games-rpg/baldurs-gate-ee/baldurs-gate-ee-2.6.6.0.47291.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop prefix unpacker wrapper
+
+DESCRIPTION="Baldur's Gate: Enhanced Edition"
+HOMEPAGE="https://www.baldursgate.com/"
```

Comment:
```
RedirectedUrl: version 2.6.6.0.47291: HOMEPAGE: permanently redirected: https://www.baldursgate.com/ -> https://dnd.wizards.com/products/baldurs-gate
```

Reply:
I will change this to `https://baldursgate.beamdog.com/` as it more accurately refers to the games homepage rather than the collection of Baldur's Gate games.